tests/main/interfaces-system-dbus/task.yaml

```diff
@@ -0,0 +1,37 @@
+summary: Ensure that the dbus interface works on the system bus.
+
+execute: |
+    #shellcheck source=tests/lib/dirs.sh
+    . "$TESTSLIB/dirs.sh"
+
+    echo "Install dbus system test snaps"
+    snap install --edge test-snapd-dbus-provider
```

Comment:
you could install it from beta

Reply:
Indeed, soon I will be able to do that :) right now the system dbus bits are only available in edge but once things are all green I will promote.